main.go

```diff
@@ -194,12 +194,6 @@ func setupReconcilersHub(mgr ctrl.Manager) {
 		os.Exit(1)
 	}
 
-	// setup webhook for DRPolicy
-	if err := (&ramendrv1alpha1.DRPolicy{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "DRPolicy")
-		os.Exit(1)
-	}
-
 	// setup webhook for drpc
```

Comment:
Can we do the same for drpc?

Reply:
yes, that is the plan. I am planning to send different PRs for each CR, it's just that removal of webhook code is more than adding CEL so, I plan to take one CR at a time so that we don't break ramen. :)